tests/test_datasource.py

```diff
@@ -144,7 +144,7 @@ def test_s3(s3_endpoint_url):
 
 def test_basic_excel(path):
     """It should not add a __sheet__ column when retrieving a single sheet"""
-    ds = DataSource(path("fixture-multi-sheet.xlsx"))
+    ds = DataSource(path("fixture-multi-sheet.xlsx"), reader_kwargs={"sheet_name": "January"})
```

Comment:
You should NOT change tests. We want backward compatibility

Reply:
For this specific case i think there is a typo or an rror with th file we're loading then !
In the case of the `test_basic_excel` we should load `fixture-single-sheet.xlsx` that has a single sheet !
I updated that here : https://github.com/ToucanToco/peakina/pull/69/commits/559e13c8c03fe51d06ac377905196b0dff245922

Let me know if we're in the same line !